src/pages/Login/login.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import Cookies from 'js-cookie';
+import { Link, useNavigate } from 'react-router-dom';
+import { AiFillExclamationCircle} from 'react-icons/ai';
+import showToast from 'utils/toastMessage';
+import { loginSchema } from 'utils/validationSchema';
+import { IApiResponse, ILogin } from 'utils/interface';
+import { loginUser } from 'api/user';
+import { CONSTANTS } from 'utils/constant';
+import './style.css';
```

Comment:
can we separate internal imports by empty line by other imports + remove relative imports